great_expectations/rule_based_profiler/config/base.py

```diff
@@ -99,7 +99,7 @@ def __init__(
         self.batch_request = batch_request
         for k, v in kwargs.items():
             setattr(self, k, v)
-            logger.warning(
+            logger.debug(
                 "Setting unknown kwarg (%s, %s) provided to config as attr", k, v
             )
```

Comment:
Between numerous logging statements and metric calc progress bars, the notebook is super difficult to decipher. Let's keep things simple.